x-pack/plugins/security_solution/public/endpoint_hosts/view/details/policy_response.tsx

```diff
@@ -136,8 +136,14 @@ export const PolicyResponse = memo(
           const attentionCount = responseAttentionCount.get(key);
           return (
             <PolicyResponseConfigAccordion
-              id={useMemo(() => htmlIdGenerator()(), [])}
-              key={useMemo(() => htmlIdGenerator()(), [])}
+              id={
+                /* eslint-disable-next-line react-hooks/rules-of-hooks */
```

Comment:
what is the rule here? That all hooks be called from the root of the component and its value referenced here?

Reply:
![image](https://user-images.githubusercontent.com/35559/83805180-642cc580-a67d-11ea-8b24-c96ffb238979.png)
![image](https://user-images.githubusercontent.com/35559/83805193-6abb3d00-a67d-11ea-9bc6-c809d33aba89.png)

Same for the other one next to it. The hook call is inside `Object.entries`